tests/karate/src/test/java/be/utils/ScalaServer.java

```diff
@@ -17,7 +25,7 @@ public void stop() {
   @Override
   public String[] getCmd() {
     String configPath = Paths.get("src", "main", "scala", "ch", "epfl", "pop", "config").toString();
-    return new String[]{"sbt", "-Dscala.config=" + configPath, "run"};
+    return new String[] { "sbt.bat", "-Dscala.config=" + configPath, "run" };
```

Comment:
Isn't that windows specific ?

Are you sure we can run this in a linux environment ?

Reply:
Agreed, this (most likely) won't work in a linux environment.